tests/test_conference.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import datetime
+import kenpompy.conference as kpconf
+import pandas as pd
+
+def test_get_valid_conferences(browser):
+	expected = 32
+
+	confs_2021 = kpconf.get_valid_conferences(browser, season = '2021')
+	assert len(confs_2021) == expected
+
+	valid_2021_confs = ['B12', 'P12', 'Pat']
+	for team in valid_2021_confs:
+		assert team in confs_2021
+
+	confs_2003 = kpconf.get_valid_conferences(browser, season = '2003')
+	assert len(confs_2003) == 32
+
+	valid_2003_confs = ['MEAC', 'CAA']
+	for team in valid_2003_confs:
+		assert team in confs_2003
+
+def test_get_aggregate_stats(browser):
+	expectedTempo = 67.9
+	expectedTempoRank = 20
+	expectedHomeWinPercent = 60.3
+	expectedHomeWinPercentRank = 8
+
+
+	confs_2021 = kpconf.get_aggregate_stats(browser, 'B10', season = '2021')
+	assert confs_2021.loc['Tempo', 'Value'] == expectedTempo
+	assert confs_2021.loc['Tempo', 'Rank'] == expectedTempoRank
+	assert confs_2021.loc['Home win%', 'Value'] == expectedHomeWinPercent
+	assert confs_2021.loc['Home win%', 'Rank'] == expectedHomeWinPercentRank
+	
+
```

Comment:
Another nitpick, can you tighten up the spacing in these and have two empty lines between functions? Otherwise looks good to me, thanks for the PR!